Makefile.common

```diff
@@ -183,7 +183,9 @@ HASH_BUNDLE= \
 # therefore Hacl.Impl.SHA3 cannot be put on the right-hand side of the bundle
 # (which would eliminate internal helpers not used otherwise, such as absorb or
 # squeeze)
-SHA3_BUNDLE=-bundle Hacl.Streaming.Keccak+Hacl.SHA3+Hacl.Impl.SHA3=Hacl.Hash.SHA3[rename=Hacl_Hash_SHA3,rename-prefix]
+SHA3_BUNDLE=-bundle Hacl.Streaming.Keccak+Hacl.SHA3+Hacl.Impl.SHA3=Hacl.Hash.SHA3[rename=Hacl_Hash_SHA3]
+SHA3_SCALAR_BUNDLE=-bundle Hacl.Hash.SHA3.Scalar=Hacl.Spec.SHA3.Vec.*,Hacl.Impl.SHA3.Vec,Hacl.Hash.SHA3.Scalar
```

Comment:
when `Hacl.Hash.SHA3.Scalar` is already on the left-hand side of the `=`, you do not need to repeat it on the right-hand side, so please remove `Hacl.Hash.SHA3.Scalar` at the end of the line... same for the line below

Reply:
Removed both `Hacl.Hash.SHA3.Scalar` and `Hacl.Hash.SHA3.Simd256` on the right-hand.